libs/core/src/lib/decorators/property.ts

```diff
@@ -80,6 +80,25 @@ export interface IPropertyOptions
  * public volume: number = 0; // declaration of the property
  * ```
  * _**Note**_: the name `volume` is in camelCase as the naming convention.
+ * 
+ * With Player version 8.0.2 and after, you can create your own getter and setter for the property. This is really helpful when you want some logic if the value of the property change.
+ * To do that, you have to declare your property with '_' (_volume)
+ * Then create a getter and setter like this :
+ * 
+ * ```ts
+ * public get volume(): number
+ * {
+ *      return this._volume;
+ * } 
+ * public set volume(newVolume: number)
+ * {
+ *      if(newVolume !== this._volume)
+ *      {
+ *          this._volume = newVolume;
+ *          // Your logic here ... 
+ *      }
+ * }
+ * ```
```

Comment:
Just a more detailed example
```suggestion
 * **With Player version 8.0.2 and after**, you can create your own getter and setter for the property. This is really helpful when you want some logic if the value of the property change.
 * To do that, you have to declare your property prefixed by `_` (i.e.: `_volume`)
 * Then create a getter and setter like this :
 * 
 * ```ts
 * @Property({
 *     displayName: 'Volume', // 'Volume' is the name of the property
 *     description: 'Current volume in the media.', // here the description of the property
 *     defaultValue: 1, // the default value of the property : 1
 *     minValue: 0, // the minimum value : 0
 *     maxValue: 1, // the maximum value : 1
 *     type: Number // the property is a number (so binding on a text with '0.5' value will be converted in a number value 0.5)
 * })
 * public _volume: number = 0;
 *
 * public get volume(): number
 * {
 *      return this._volume;
 * } 
 * public set volume(newVolume: number)
 * {
 *      if(newVolume !== this._volume)
 *      {
 *          this._volume = newVolume;
 *          // Put your code here and preferably use `this._volume` instead of `newVolume` as internal Player logic will have apply type conversion and custom range value on it.
 *      }
 * }
 * ```
```